### DISCUSSION TITLE
How to get the window size in WebAssembly?

### DISCUSSION THREAD

**Discussion by aizcutei** [emilk/egui#2112] [Q&A] [answered]
Hi I am using the eframe as the backend and hope to run both on native and wasm. I want to get the window size and I found it is easy on native by using `frame.info().window_info.size`, but when I was testing under was, I got an error said `no field 'window_size' on type 'IntegrationInfo'`. So is there a way to get the browser size?

**Comment by KentaTheBugMaker** [comment, marked as answer]
use web_sys canvas api
```rust

let canvas_id ="the_canvas_id";// may differ your page configuration.
let canvas = eframe::canvas_element(canvas_id).unwrap();
let width = canvas.width();
let height = canvas.height();
```

**Reply by aizcutei** [reply]
Thank you! But seems `eframe::canvas_element` cannot access. I choose to use raw `web_sys` to do that.
